webpack.pro.config.js

```diff
@@ -0,0 +1,25 @@
+const path = require('path');
+
+module.exports = {
+  entry: './client/index.jsx',
+  output: {
+    filename: 'app.js',
+    path: path.resolve(__dirname, 'public'),
+  },
+  mode: 'development',
```

Comment:
Hey Elvin, a few questions: Does this webpack config file completely replace the webpack config you used during development? Or does it supplement it (i.e. you have two webpack configs, one for production and one for development)? If you do have two webpack configs, what are the differences?

If this is a production webpack, shouldn't the mode be set to "production"?
If you have two webpack config files, shouldn't you also have two build scripts? The SDC team that picks our component may need an easy way to use your webpack config for development.

Reply:
Great questions, Andrew. Yes, the production webpack should be in "production" mode. I am fixing that right now. 

And this file is in addition to the dev version. The only difference is that the "watch" property is set to false (in dev version it's set to true). There are separate scripts that run both versions:
- "npm run client-dev" uses the dev webpack and continuously rebuilds /public/app.js as changes in client source code are made. 
- "npm run build" uses the production webpack and rebuilds /public/app.js only once from the client source.